pkg/kmodule/kmodule_linux.go

```diff
@@ -174,7 +174,15 @@ func genDeps() (depMap, error) {
 	}
 	rel := string(u.Release[:bytes.IndexByte(u.Release[:], 0)])
 
-	moduleDir := filepath.Join("/lib/modules", strings.TrimSpace(rel))
+	moduleDirs := []string{"/lib/modules", "/usr/lib/modules"}
+
+	var moduleDir string
+	for _, moduleDirs := range moduleDirs {
```

Comment:
you could also do an anon variable if you want
for _, moduleDirs := range []string{"/lib/modules", "/usr/lib/modules"} {

which is kind of nice as it avoids two variables same name. 
Also, for compactness, it's typical to have the for name be something a bit tidier:
for _, n := range []string{"/lib/modules", "/usr/lib/modules"} {
...
}